src/plugins/intel_npu/src/backend/include/zero_init.hpp

```diff
@@ -56,13 +47,46 @@ class ZeroInitStructsHolder final {
         return mutable_command_list_version;
     }
 
+
+    static ze_context_handle_t& getContext() {
+        return context;
+    }
+
+    static void setContext(ze_context_handle_t &newContext)  {
+        context=newContext;
+    }
+
+    static ze_driver_handle_t& getDriverHandle() {
```

Comment:
i have revert this change, and use the previous implementation
`inline ze_driver_handle_t getDriver() const` instead of `static ze_driver_handle_t& getDriverHandle()`